src/meta/duplication/meta_duplication_service.cpp

```diff
@@ -308,11 +310,66 @@ void meta_duplication_service::duplication_sync(duplication_sync_rpc rpc)
     }
 }
 
-// todo(jiashuo1) wait detail implementation
 void meta_duplication_service::trigger_follower_duplicate_checkpoint(
     const std::shared_ptr<duplication_info> &dup, const std::shared_ptr<app_state> &app)
 {
-    dup->alter_status(duplication_status::DS_APP);
+    configuration_create_app_request request;
+    request.app_name = app->app_name;
+    request.options.app_type = app->app_type;
+    request.options.partition_count = app->partition_count;
+    request.options.replica_count = app->max_replica_count;
+    request.options.success_if_exist = true;
+    request.options.envs = app->envs;
+    request.options.is_stateful = app->is_stateful;
+
+    duplication_options opts;
+    opts.app_name = app->app_name;
+    opts.cluster_name = get_current_cluster_name();
+    opts.metas = _meta_svc->_opts.meta_servers;
+    request.options.__set_duplication(opts);
+
+    request.options.envs.emplace(duplication_constants::kDuplicationMasterFlag,
```

Comment:
It seems that follower table will store master meta address list in app_envs, please adding comments for it.